config/routes.rb

```diff
@@ -6,8 +6,21 @@
   resources :jukes
   get 'invite_code' => 'jukes#invite_code'
   get 'join' => 'jukes#join'
-  resources :users
-  resources :sessions, only: [:create, :destroy]
+  resources :playlists, only: [:index, :show, :create]
+
+  resources :users do
+    resources :playlists, only: [:index, :show, :create] do
```

Comment:
don't nest these, we don't need all of these routes.  we don't need to nest act all probably.